src/Analysis/GWASV2/Components/Diagrams/CohortsOverlapDiagram/CohortsOverlapDiagram.jsx

```diff
@@ -142,12 +144,12 @@ const CohortsOverlapDiagram = ({
     set1Size: selectedStudyPopulationCohort.size,
     set2Size: selectedCaseCohort.size,
     set3Size: selectedControlCohort.size,
-    set12Size: dataPopCase.cohort_overlap.overlap_after_filter,
-    set13Size: dataPopControl.cohort_overlap.overlap_after_filter,
-    set23Size: dataCaseControl.cohort_overlap.overlap_after_filter,
+    set12Size: dataPopCase.cohort_overlap.case_control_overlap,
+    set13Size: dataPopControl.cohort_overlap.case_control_overlap,
+    set23Size: dataCaseControl.cohort_overlap.case_control_overlap,
     set123Size:
-      dataPopCase.cohort_overlap.overlap_after_filter
-      - dataPopCaseMinusPopCaseControl.cohort_overlap.overlap_after_filter,
+      dataPopCase.cohort_overlap.case_control_overlap
+      - dataPopCaseMinusPopCaseControl.cohort_overlap.case_control_overlap,
```

Comment:
Question: I'm not familiar with the library: are there any potential issues that could arise with a very large or very small number as a result of the subtraction?

Reply:
`dataPopCaseMinusPopCaseControl` is a subset of `dataPopCase`, so in this case there should be no issues. We could let the overlap diagram throw an error in case of negative set sizes...that would make it a bit more defensive.